### ISSUE TITLE
Rpgle Indentation Problem

### ISSUE THREAD

**Issue by pabloto** [halcyon-tech/vscode-ibmi#397]
**Describe the bug**
Hi all,

the compiler didn't accept an End-Ds if i declare a Dcl-Ds LikeRec(Myrcd : *All);
starting from this line the indentation control give me all indentation error.

Look at this:

![image](https://user-images.githubusercontent.com/9719313/141780626-9c3ffcc9-f44c-4310-aabf-14879b776b47.png)

**Comment by worksofliam** [comment]
@pabloto Yes, once you disable you will need to restart the editor - hoping to change this in the future.

Also, can you share the snippet of code that I can copy and paste to recreate it? It should be easy to solve.

**Comment by pabloto** [comment]
Hi Liam,

here's a snippet:

`**free
Ctl-Opt  DftActGrp(*no)
  ActGrp(*StgMdl)
  StgMdl(*SNGLVL)
  Expropts(*AlwBlankNum)
  Option(*NoUnRef :*SrcStmt :*NoDebugIo)
  DatFmt(*Iso) TimFmt(*Iso)
  Debug(*Constants)
  AlwNull(*UsrCtl)
  DftName(TSTLIKEREC)
  Text('Test Ds LikeRec');

Dcl-F Employee Keyed ExtFile('SAMPLE/EMPLOYEE') Qualified;

Dcl-Ds DsEmployee LikeRec(Employee.Employee :*ALL);

// ____________________________________________________________________________

Chain '000010' Employee DsEmployee;
If Not %found(Employee);
  Return;
EndIf;

Dsply DsEmployee.FirstNme;

Return;

`

**Comment by SJLennon** [comment]
Similar to #342?
@worksofliam @pabloto

**Comment by worksofliam** [comment]
@SJLennon Yup. Very similar indeed. Easy fix.
